rero_ils/config.py

```diff
@@ -1586,18 +1588,20 @@ def _(x):
 }
 
 #: Invenio circulation configuration.
-CIRCULATION_ITEM_EXISTS = Item.get_record_by_pid
+CIRCULATION_ITEM_EXISTS = item_exists
```

Comment:
My point of view : I thing the best place for this method is to place it into the `modules.items.api:Item` class as a static method. Maybe other comments to confirm of not this point of view.